aws/auto-deployment.yaml

```diff
@@ -0,0 +1,232 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Auto deployment of Logz.io AWS Cost and Usage.
+Parameters:
+  S3BucketName:
+    Type: String
+    Description: >-
+      The name for the bucket which will contain the report files. The bucket name
+      must contain only lowercase letters, numbers, periods (.), and dashes (-),
+      and must follow Amazon S3 bucket restrictions and limitations.
+    Default: my-s3-bucket
+    AllowedPattern: '^[a-z0-9\-\.]*$'
+    MinLength: 1
+  ReportName:
+    Type: String
+    Description: >-
+      The name of report that you want to create. The name must be unique, is
+      case sensitive and can't include spaces.
+    Default: my-report
+    AllowedPattern: ^\S*$
+    MinLength: 1
+  ReportPrefix:
+    Type: String
+    Description: >-
+      The prefix that AWS adds to the report name when AWS delivers the report.
+      Your prefix can't include spaces.
+    Default: reports
+    AllowedPattern: ^\S*$
+    MinLength: 1
+  ReportTimeUnit:
+    Type: String
+    Description: >-
+      The granularity of the line items in the report. (Enabling hourly reports
+      does not mean that a new report is generated every hour. It means that
+      data in the report is aggregated with a granularity of one hour.)
+    Default: DAILY
+    AllowedValues:
+      - HOURLY
+      - DAILY
+      - MONTHLY
+  ReportAdditionalSchemaElements:
+    Type: String
+    Description: >-
+      Choose INCLUDE if you want AWS to include additional details
+      about individual resources IDs in the report (This might significantly
+      increase report size and might affect performance. AWS Lambda can run for
+      up to 15 minutes with up to 10240 MB, and the process time for the whole
+      file must end within this timeframe.), or DON'T INCLUDE otherwise.
+    Default: DON'T INCLUDE
+    AllowedValues:
+      - DON'T INCLUDE
+      - INCLUDE
+  LogzioURL:
+    Type: String
+    Description: >-
+      The Logz.io listener URL. If you are in the EU region choose
+      https://listener-eu.logz.io:8071. Otherwise, choose
+      https://listener.logz.io:8071. (You can tell which region you are in by
+      checking your login URL - app.logz.io means you are in the US, and
+      app-eu.logz.io means you are in the EU.)
+    Default: 'https://listener.logz.io:8071'
+    AllowedValues:
+      - 'https://listener.logz.io:8071'
+      - 'https://listener-eu.logz.io:8071'
+  LogzioToken:
+    Type: String
+    Description: >-
+      Your Logz.io account token. (Can be retrieved on the Settings page in the
```

Comment:
Should be
Your Logz.io logs token ( can be retrieved from the Manage Token page)

Reply:
Done